main/src/modules/abm/autobiographicalMemory/src/autobiographicalMemory.cpp

```diff
@@ -446,6 +446,8 @@ bool autobiographicalMemory::respond(const Bottle& bCommand, Bottle& bReply)
 {
     Bottle bError;
     bReply.clear();
+    bReply.addVocab(Vocab::encode("many"));
```

Comment:
Same as in `abmReasoning` - not sure what this is used for. However, I am quite concerned that the change here breaks quite a few other modules.